sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -604,17 +734,56 @@ class RocksDB(
   }
 
   private def getDBProperty(property: String): Long = {
-    db.getProperty(property).toLong
+    if (useColumnFamilies) {
+      // get cumulative sum across all available column families
+      assert(!colFamilyNameToHandleMap.isEmpty)
+      colFamilyNameToHandleMap
+        .values
+        .map(handle => db.getProperty(handle, property).toLong)
+        .sum
```

Comment:
Maybe we should have done the check as it should be existing issue if there is any since we accumulate values from all partitions. But good to double check, and probably leave a code comment (even in current codebase).

Reply:
This is actually only an issue only for multiple col families at this layer - upper layers will accumulate across partitions